helper-cli/src/main/kotlin/commands/packagecuration/SplitCommand.kt

```diff
@@ -57,38 +56,38 @@ internal class SplitCommand : CliktCommand(
         .required()
 
     override fun run() {
-        if (!outputCurationsDir.list().isNullOrEmpty()) {
-            throw UsageError("Output directory $outputCurationsDir must be empty.")
-        }
-
         val packageCurations = inputCurationsFile.readValue<List<PackageCuration>>()
         val groupedCurations = packageCurations.groupBy {
             getSplitCurationFile(outputCurationsDir, it.id, inputCurationsFile.extension)
         }
 
         val mapper = createBlockYamlMapper()
         groupedCurations.forEach { (outputFile, curations) ->
-            curations.forEach { originalCuration ->
+            val curationsToPersist = if (outputFile.isFile) {
+                outputFile.readValue<Set<PackageCuration>>()
+            } else {
+                emptySet()
+            }.toMutableSet()
```

Comment:
Could be

            val curationsToPersist = if (outputFile.isFile) {
                outputFile.readValue<MutableSet<PackageCuration>>()
            } else {
                mutableSetOf()
            }

to avoid the final set conversion.